Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs

```diff
@@ -90,6 +92,46 @@ public float FollowHandCameraFacingThresholdAngle
             set => followHandCameraFacingThresholdAngle = value;
         }
 
+        [SerializeField]
+        [Tooltip("With this active, solver will activate the UI after the palm threshold has been met and the user gazes at the activation point")]
+        private bool useGazeActivation = false;
+
+        /// <summary> 
+        /// With this active, solver will activate after the palm threshold has been met and the user gazes at the activation point
+        /// </summary>
+        public bool UseGazeActivation
+        {
+            get => useGazeActivation;
+            set => useGazeActivation = value;
+        }
+
+        [SerializeField]
+        [Tooltip("The distance between the planar intersection of the eye gaze ray and the activation transform. Uses square magnitude between two points for distance")]
+        [Range(0.0f, .1f)]
+        private float gazeProximityThreshold = .005f;
```

Comment:
5mm is an extremely narrow threshold if using the headgaze rather than the eye gaze.

Reply:
That's fair; given the dual utility of this I will likely need to adjust the threshold to be more forgiving for headgaze. I'll come up with some method of doing so.